src/odb.h

```diff
@@ -54,6 +57,10 @@ typedef enum {
  */
 int git_odb__set_caps(git_odb *odb, int caps);
 
+int git_odb__open_own(git_odb **out, git_repository *repo);
+
+int git_odb__is_default(git_odb *odb);
```

Comment:
This isn't yet used, right? What's the usecase of it?

Reply:
To know if we're running against the "default" backend stack, when only loose/packed backends are present. Not sure if it's useful as well…

Reply:
Then let's keep it out for now, ok? Not opposed to adding this at all, but I really would like to merge this PR and discussion of those things might put it on hold longer than needed.